src/main/java/hudson/plugins/locale/PluginImpl.java

```diff
@@ -134,6 +137,13 @@ public GlobalConfigurationCategory getCategory() {
         return GlobalConfigurationCategory.get(AppearanceCategory.class);
     }
 
+    public List<Locale> getLocales() {
+        List<Locale> locales = new ArrayList<>();
+        Locale[] availableLocales = Locale.getAvailableLocales();
```

Comment:
This misses to add the possibility to leave the locale empty -> I don't want to set a locale but use what my browser tells

Reply:
By default, the locale is set as the user's browser's default locale. The user can change it if he wants to. 

(or) Do you mean, I need to have an option such as "Default/Auto" in the dropdown? That's a good point, because if the user has changed it and wants to revert it, then he can do it via that option.

Reply:
yes it must be possible to reset to the default.